input/pagecontent/case-studies-english.md

```diff
@@ -1,88 +1,49 @@
-
 * Establishment (name, address, establishment number)
 * Number of employees for biological monitoring
 * Examinations to be carried out (e.g. arsenic, lead, mercury, mandelic acid). The companies receive a list from the AMV including labels with the affected employees. Here we take the document of an employee Beat Borer, born 6. 6. 1986. The laboratory prints the order documents with the company number and sends the necessary number of urine cups and order documents to the companies. In the company, order documents and urine cups are labelled with the labels supplied by the AMV and given to the employees. The samples, including the order document, are sent from the company to the laboratory.
 
-[Biological monitoring SUVA](https://www.suva.ch/de-CH/material/Factsheets/biologisches-monitoring-und-biologische-arbeitsstofftoleranzwerte)
-
-[Order Document with ServiceRequest](Bundle-5-biol-monit-document-with-sr.html)
-
-[Order Document with ServiceRequest and Form](Bundle-5-biol-monit-document-with-sr-and-form.html)
-
-### CH LAB-Order 6-histopath
-
-#### Suspected Melanoma, order for histopathology examination case study
-
-A 45-year-old female patient Simone Studer came for her regular skin check to the Arztpraxis of Dermatologist Armin Ahrens, and this lesion was noted: Junctions nevi consisting of seborrheic area top end and atypical network constructed of many seperate dots, structures , etc , suggesting melanocytic involvement. Excision. Attachement of dermatoscopic images Specimen: 29 x 11 x 5 mm skin ellipse Body Site:
+* [CH LAB-Order Document with Service Request](Bundle-5-biol-monit-document-with-sr.html)
+* [CH LAB-Order Document with Service Request and Form](Bundle-5-biol-monit-document-with-sr-and-form.html)
 
+### Histopathological Examination
+A 45-year-old female patient Simone Studer came for her regular skin check to the Arztpraxis of Dermatologist Armin Ahrens, and this lesion was noted: Junctions nevi consisting of seborrheic area top end and atypical network constructed of many separate dots, structures, etc., suggesting melanocytic involvement. Excision. Attachment of dermatoscopic images Specimen: 29 x 11 x 5 mm skin ellipse Body Site:    
 * Back paralumbar left
 * The skin sample, including the order document and the dermatoscopic image are sent to the laboratory.
 
-[Order Document with ServiceRequest](Bundle-6-histopath-document-with-sr.html)
-
-[Order Document with ServiceRequest and Form](Bundle-6-histopath-document-with-sr-and-form.html)
-
-{% include img.html img="Dermatoscopic-image-of-superficial-spreading-melanoma.png" caption="Fig. 3: Dermatoscopic Image" width="30%" %}
```

Comment:
sollte das image wirklick weg? sehe Dermatoscopic-image-of-superficial-spreading-melanoma.png nicht entfernt im PR